export/orbax/export/jax_module.py

```diff
@@ -186,7 +186,7 @@ def _get_param_names(params: PyTree) -> PyTree:
   """Gets parameter names for PyTree elements."""
 
   def _param_name_from_keypath(keypath: Tuple[Any, ...]) -> str:
-    return '.'.join([str(ckpt_utils.get_key_name(k)) for k in keypath])
+    return '.'.join([str(ckpt_utils.get_key_name(k)) for k in keypath]).replace("~", "_")
```

Comment:
Please use `'` instead of `"` (be consistent with the file other places).  Also this line is long than 80.

Reply:
Done